package.json

```
@@ -80,9 +80,9 @@
     "@mui/internal-markdown": "^1.0.3",
     "@mui/internal-scripts": "^1.0.6",
     "@mui/internal-test-utils": "https://pkg.csb.dev/mui/material-ui/commit/fb183624/@mui/internal-test-utils",
-    "@mui/material": "5.15.15",
+    "@mui/material": "6.0.0-alpha.4",
     "@mui/monorepo": "github:mui/material-ui#09630ee4f9fcbe7a9765d586355b7c54dbe7ad85",
```

Let's update the monorepo to the version consistent with the packages to avoid potential weird errors
```suggestion
    "@mui/monorepo": "github:mui/material-ui#v6.0.0-alpha.4",
```